docker-compose.yaml

```diff
@@ -2,16 +2,17 @@
 services:
   node:
     container_name: sample_validator
-    image: "alloranetwork/allora-chain:v0.2.14"
+    image: "alloranetwork/allora-chain:v0.3.1" # docker image without cosmovisor. use vx.x.x-upgrader for upgrade image
```

Comment:
Shouldn't be v0.3.0?

Reply:
so people are already using that v0.3.1 tag. I don't want to overide it such that it could break already running services

Reply:
changed to v0.3.0 actually. we are only releasing the upgrader